t/config.t

```diff
@@ -154,7 +154,7 @@ subtest 'Test configuration override from file' => sub {
 
     ok -e $t_dir->child("openqa.ini");
     ok($app->config->{global}->{suse_mirror} eq 'http://blah/',   'suse mirror');
-    ok($app->config->{audit}->{blacklist} eq 'job_grab job_done', 'audit blacklist');
+    ok($app->config->{audit}->{blocklist} eq 'job_grab job_done', 'audit blocklist');
     is($app->config->{'assets/storage_duration'}->{'-CURRENT'}, 40, 'assets/storage_duration');
```

Comment:
We should have a test to see if the old setting still works

Reply:
Done that now.